test/tf_exec.cpp

```diff
@@ -325,6 +325,30 @@ TEST(tf_exec, Op_Square) {
   EXPECT_FLOAT_EQ(0.0, mat(1, 1));
 }
 
+TEST(tf_exec, Op_SquaredDifference) {
+  tf::Scope root = tf::Scope::NewRootScope();
+  tf::Scope root_ngraph = root.NewSubScope("sub_scope_ngraph");
+  root_ngraph = root_ngraph.WithDevice("/device:NGRAPH:0");
+
+  auto A = tf::ops::Placeholder(root, tf::DataType::DT_FLOAT);
+  auto B = tf::ops::Placeholder(root, tf::DataType::DT_FLOAT);
+  auto r = tf::ops::SquaredDifference(root_ngraph.WithOpName("r"), A, B);
+
+  std::vector<tf::Tensor> outputs;
+  tf::ClientSession session(root);
+
+  TF_CHECK_OK(session.Run({{A, {{3.f, 5.f}, {2.f, 0.f}}}, {B, {{1.f, 2.f}, {-1.f, 1.f}}}}, {r}, &outputs)); 
+  ASSERT_EQ(outputs[0].shape(), tf::TensorShape({2,2}));
+
+  auto mat = outputs[0].matrix<float>();
+  EXPECT_FLOAT_EQ(4.0, mat(0, 0));
+  EXPECT_FLOAT_EQ(9.0, mat(0, 1));
+  EXPECT_FLOAT_EQ(9.0, mat(1, 0));
+  EXPECT_FLOAT_EQ(1.0, mat(1, 1));
+
```

Comment:
Are there any corner cases we need to test here? Probably not but just checking.

Reply:
I have passed other numbers and done some sanity checks for this Op. Nothing that needs to be here.